lib/v2/ccreports/index.js

```diff
@@ -0,0 +1,40 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+const rootUrl = 'https://www.ccreports.com.cn';
+
+module.exports = async (ctx) => {
+    const listData = await got.get(rootUrl);
+    const $ = cheerio.load(listData.data);
+    const list = $('div.index-four-content > div.article-box')
+        .find('div.new-child')
+        .map((_, item) => ({
+            title: $(item).find('p.new-title').text(),
+            link: new URL($(item).find('a').attr('href'), rootUrl).href,
+            author: $(item)
+                .find('p.new-desc')
+                .text()
+                .match(/作者：(.*?)\s/)[1],
+        }))
+        .get();
+
+    const items = await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const detailData = await got.get(item.link);
+                const $ = cheerio.load(detailData.data);
+                item.description = $('div.pdbox').html();
+                item.pubDate = parseDate($('div.newbox > div.newtit > p').text(), 'YYYY-MM-DD HH:mm:ss');
```

Comment:
You need to adjust the timezone also. See Vercel's output vs the website.

Reply:
Done.